packages/common/components/content/blocks/page-body.marko

```diff
@@ -32,4 +32,7 @@ $ const displayMedia = displayPrimaryImage || content.embedCode;
     </div>
   </if>
   <cms-content-body tag="div" block=block obj=input.content />
+  <if(input.content.type === "whitepaper")>
```

Comment:
Check against `content.type` - otherwise its possible to get an undefined error.